docs/quickstarts/nextjs.mdx

```diff
@@ -29,7 +53,11 @@ Once you have a Next.js application ready, you need to install Clerk's Next.js S
 
 ### Set environment keys
 
-Below is an example of an `.env.local` file. If you are signed into your Clerk Dashboard, your keys should become visible by clicking on the eye icon.
+In your Next.js project's root folder, you may have an `.env.local` file alongside `package.json` and other configuration files. If you don't see it, create it.
+
+Add the following code to your `.env.local` file to set your public and secret keys.
+
+**Pro tip!** If you are signed into your [Clerk Dashboard](https://dashboard.clerk.com/), your keys should become visible by clicking on the eye icon.

```

Comment:
I agree. Usually if there is a "tip", it's good opportunity to use a callout with type="info".

Reply:
I chose not to use a callout for this because it felt unworthy of being a callout. This page previously looked like a patchwork of callouts, and there can be a tendency in docs to "make everything a call out." Call outs are, by their nature, disruptive to reading. I didn't think this was worth disrupting their flow. Perhaps marrying it into the preceding content is a better idea?